Library/ExperimentName+HelpersTests.swift

```diff
@@ -37,7 +42,7 @@ final class ExperimentName_HelpersTests: TestCase {
     let config = Config.template
       |> \.abExperiments .~ ["unknown": "experimental"]
 
-    withEnvironment(config: config) {
+    withEnvironment(config: config, mainBundle: self.releaseBundle) {
```

Comment:
Can we add a test to verify that when the bundle *is not* the `.isRelease` bundle, that the `debugDefault` is returned?